### PR TITLE
COMMERCE-10305 Improve tests BulkDeleteOrders and BulkDeleteCountries

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-10305

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceSettings.testcase [hunk 1]
```diff
@@ -19,21 +19,46 @@ definition {
...
-		AssertElementNotPresent(locator1 = "CommerceEntry#ROW_VERTICAL_ELLIPSIS");
-	}
+		task ("And the admin can delete all the orders with one click by using the select all checkbox"){
+			CommerceNavigator.searchEntry(entryName = "Qatar");
+
+			AssertElementNotPresent(
+					key_entryName = "Qatar",
+					locator1 = "CommerceNavigation#ENTRY_NAME");
+
+			CommerceNavigator.searchEntry(entryName = "");
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceSettings.testcase:56]: Can you put these lines in another task?
Here I think you are checking if the entry Qatar is still present or not.
Thanks!

**Review by pietrobocale — CHANGES_REQUESTED** [review]

**Comment by pietrobocale** [comment]
Can you change this commit "COMMERCE-10305 BulkDeleteOrders & BulkDeleteCountries SF" in "COMMERCE-10305 Auto SF"
Thanks!

**Comment by pietrobocale** [comment]
Remove the DO NOT MERGE if this PR is the one that will be forwarded
Thanks!

**Comment by pietrobocale** [comment]
Review completed, changes requested

**Comment by Brunno0liveira** [comment]
Changes made

Thanks!

**Comment by pietrobocale** [comment]
Review Passed

**Comment by pietrobocale** [comment]
ci:forward

**Pull request closed without merging**